### DISCUSSION TITLE
Clear

### DISCUSSION THREAD

**Discussion by peteringram0** [zellij-org/zellij#2584] [Q&A]
Using CMD+K in iterm clears out the terminal. When doing this with Zellij it clears off the borders etc which is not ideal. How do i clear a single pane?

**Comment by Lockszmith-GH** [comment]
What OS? What Terminal?

In a POSIX shell, runnig the `clear` command should clear the screen, which clears the Zellij Pane.

if you check the default keybindings, `Alt`+`K` is equivalent to `Alt`+`Up`.

So what you are describing might be a feature of your Terminal that you can disable.

**Reply by peteringram0** [reply]
This is on OSX with Iterm2. Alt+Up scrolls the terminal window up. Running `clear` clears the terminal but scrolling up you can see the previous commands. Ideally i want a way to clear it so scrolling back up wont show the history. In my current work flow i normally use this for tests and if its not cleared, when scrolling up its easy to mistake previous test results if they are still in the buffer

**Reply by Lockszmith-GH** [reply]
yeah, what you are describing is how it works for me in Linux with Alacritty - clear resets the history and scrolling up doesn't show previous session content.

If this is reproducable on another terminal, I would suggest to open a new issue.

**Reply by peteringram0** [reply]
I have done a bit more fiddling with this. So it happens on term and iterm2 on my machine. I found out that sending

```text
printf '\e[2J\e[3J\e[H'
```

does the same as CMD+K so have that being sent by iterm2 instead, only downside is i currently need to press enter manually after to finish off the command.

**Reply by Lockszmith-GH** [reply]
So strange... so you alias it a command? I wonder why `clear` doesn't work the same way.
Anyway, I'm happy to learn you found a workaround.

**Reply by NN---** [reply]
Agree, it is some inconvenience.
The workaround for me is to use search Ctrl+S and then PgUp/PgDown.
